python/tests/compat/test_mantid.py

```diff
@@ -18,8 +18,9 @@ def mantid_is_available():
         return False
 
 
-@pytest.mark.skipif(not mantid_is_available(),
-                    reason='Mantid framework is unavailable')
+# @pytest.mark.skipif(not mantid_is_available(),
+#                     reason='Mantid framework is unavailable')
+@pytest.mark.skip(reason='Mantid framework is unavailable')
```

Comment:
Should this have been committed?

Reply:
No it shouldn’t, was while I was testing stuff. Good catch!